User/j andrew/replace execute (#2423)
MicrosoftDocs/edge-developer: microsoft-edge/webdriver-chromium/ie-mode.md

> If your test code creates a new browser window using one of these methods, you may need to add a short wait operation afterwards to ensure that IEDriver has detected the new window:

Reviewer: ```suggestion
If your test code creates a new browser window using one of the following methods, you may need to add a short wait operation afterwards to ensure that IEDriver has detected the new window:
```
trying to figure out apparently garbled line that had "execute"

> - Opening a new window with [window.open](https://developer.mozilla.org/docs/Web/API/Window/open) ran in page script.

Reviewer: ```suggestion
- Opening a new window by calling [window.open](https://developer.mozilla.org/docs/Web/API/Window/open) from `<script>` in the page.
```

> - Opening a new window with the WebDriver [New Window](https://w3c.github.io/webdriver/#new-window) command.

Reviewer: ```suggestion
- Opening a new window by using the WebDriver [New Window](https://w3c.github.io/webdriver/#new-window) command.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with
